docs/releases/1.9.txt

```diff
@@ -187,6 +187,10 @@ Internationalization
   for languages which can be written in different scripts, for example Latin
   and Cyrillic (e.g. ``be@latin``).
 
+* When you run :djadmin:`compilemessages` from the root directory of your
```

Comment:
I'd say, "You can now run ... project and it will find all the message files that ..."